### PR TITLE
LKE-10450 feat(nodeGrouping): add inner stroke for node grouping

### PR DESCRIPTION
**Pull request by aminerhanemi**

### CODE REVIEW

File: src/ogma/features/nodeGrouping.ts [hunk 1]
```diff
@@ -171,6 +179,7 @@ export class NodeGroupingTransformation {
     this._ogma.events.on(
       ['transformationEnabled', 'transformationRefresh'],
       async (transformations) => {
+        console.log('transformationEnabled or transformationRefresh');
```
Comment (hsenbassam) [inline src/ogma/features/nodeGrouping.ts:182]: no need for the console I think

File: src/ogma/features/nodeGrouping.ts [hunk 2]
```diff
@@ -79,6 +79,10 @@ export class NodeGroupingTransformation {
         duration: 300,
         padding: 10
       });
+      // TODO remove setTimeout when LKE-10453 is fixed
```
Comment (hsenbassam) [inline src/ogma/features/nodeGrouping.ts:82]: Hopefully we will not forget about it, at least if possible to link Jira tickets together

Comment (aminerhanemi) [reply src/ogma/features/nodeGrouping.ts:82]: the ticket is also linked to the node grouping epic. We will close the epic only when it is fixed

**Comment by aminerhanemi** [comment]
@hsenbassam the lint error is in a file that I didn't touch. I don't know why it is counted as a new warning.
I will not try to fix it now as we are not merging to develop but just the epic branch

**Review by hsenbassam — APPROVED** [review]

**Pull request merged**